src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -58,13 +58,17 @@ private enum ImageFileMachine
         private IntPtr _unmanagedCallbacks; // array of pointers to JIT-EE interface callbacks
         private Object _keepAlive; // Keeps delegates for the callbacks alive
 
-        private ExceptionDispatchInfo _lastException;
+        [UnmanagedFunctionPointer(CallingConvention.StdCall)]
+        private delegate IntPtr JitStartupDelegate(IntPtr host);
+
+        [UnmanagedFunctionPointer(CallingConvention.StdCall)]
+        private delegate IntPtr GetJitDelegate();
 
-        [DllImport(JitLibrary, CallingConvention = CallingConvention.StdCall)] // stdcall in CoreCLR!
-        private extern static IntPtr jitStartup(IntPtr host);
+        private static IntPtr s_jitLibrary;
 
-        [DllImport(JitLibrary, CallingConvention = CallingConvention.StdCall)]
-        private extern static IntPtr getJit();
+        private static JitStartupDelegate s_jitStartup;
+        private static GetJitDelegate s_getJit;
+        private ExceptionDispatchInfo _lastException;
```

Comment:
Nit: Keep this in the block with other instance vars?

Reply:
Moved in 22nd commit.